### PR TITLE
E_CLASSROOM-200 [Bugfix] Dashboard table is overlapping

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-200

### Definition of Done
- [x] The tables are overlapping in categories learned and Friend activities

#### Main note
- 
#### Pages Affected
- http://localhost:3003

### Scenarios/ Test Cases
- [ ] Stack the cards if the screen is getting smaller
- for the recent quizzess you will not stack by 4 to 2 to 1
- for the categories learned and Friend activities you can will not stack them up to 1
- 
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
bugfix1
bugfix2

### CODE REVIEW

**Comment by geneno99** [comment]
### Approved PR #56 
- [x] Able to make dashboard as responsive in desktop screen resolution
- [x] Cards are stack into each other in small screen

dashboard

-----
- Total Test time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/pages/student/main/Dashboard/components/FriendsActivities/index.module.css [hunk 1]
```diff
@@ -53,3 +51,9 @@
 .noFriendActivitesMessage {
   margin-top: 165px;
 }
+
+@media only screen and (max-width: 1366px) {
+    .titleText{
+    margin-right: 200px;
+    }
+}
```
Comment (RunstyleX) [inline src/pages/student/main/Dashboard/components/FriendsActivities/index.module.css]:
```suggestion
@media only screen and (max-width: 1366px) {
    .titleText{
        margin-right: 200px;
    }
}
```

File: src/pages/student/main/Dashboard/components/Recent/index.module.css [hunk 2]
```diff
@@ -53,4 +51,8 @@
     font-weight: 600;
     font-size: 16px;
     text-align: left;
-}
\ No newline at end of file
+}
+
+Col.Card:nth-child(3){
+    display:none;
+ }
```
Comment (RunstyleX) [inline src/pages/student/main/Dashboard/components/Recent/index.module.css]: Please add new line here. 

Also have empty new line at the bottom

File: src/pages/student/main/Dashboard/index.module.css [hunk 3]
```diff
@@ -17,34 +17,40 @@
...
     background-color: #FFFFFF;
 }
 
 .forScroll{
     overflow: auto;
-}
\ No newline at end of file
+}
+
+
```
Comment (RunstyleX) [inline src/pages/student/main/Dashboard/index.module.css]:
```suggestion
}

```

File: src/pages/student/main/Dashboard/components/Recent/index.module.css [hunk 4]
```diff
@@ -53,4 +51,9 @@
...
+
```
Comment (RunstyleX) [inline src/pages/student/main/Dashboard/components/Recent/index.module.css:59]:
```suggestion

```

Comment (RunstyleX) [inline src/pages/student/main/Dashboard/components/Recent/index.module.css]:
```suggestion
Col.Card:nth-child(3){
    display:none;
 }

```

**Pull request merged**